File: src/main/java/me/equixz/chatmood/structure/FileReader.java
package me.equixz.chatmood.structure;

import me.equixz.chatmood.ChatMod;
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;
import java.util.ArrayList;
import java.util.List;

public class FileReader {
    public static List<String> readFiles(String readFileName) {
        String folderPath = "config/ChatMod/Files";
        Path filePath = Paths.get(folderPath, readFileName);

        try {
            return Files.readAllLines(filePath);
        } catch (IOException e) {
            ChatMod.LOGGER.info("An error occurred while reading the file: " + e.getMessage());
            return new ArrayList<>();
        }
    }

    public static boolean doesFileExist(String fileName) {
        String folderPath = "config/ChatMod/Files";

        Path filePath = Paths.get(folderPath, fileName);

        return Files.exists(filePath);
    }
}

File: src/main/java/me/equixz/chatmood/structure/FileReader.kt
package me.equixz.chatmood.structure

import me.equixz.chatmood.ChatMod
import java.io.IOException
import java.nio.file.Files
import java.nio.file.Paths

object FileReader {
    fun readFiles(readFileName: String?): List<String> {
        // Specify the folder path
        val folderPath = "config/ChatMod/Files"
        // Create a Path object for the file
        val filePath = Paths.get(folderPath, readFileName)

        try {
            // Return the list of lines
            return Files.readAllLines(filePath)
        } catch (e: IOException) {
            ChatMod.LOGGER.info("An error occurred while reading the file: " + e.message)
            return listOf() // Return an empty list or handle appropriately
        }
    }

    fun doesFileExist(fileName: String?): Boolean {
        // Specify the folder path
        val folderPath = "config/ChatMod/Files"

        // Create a Path object for the file
        val filePath = Paths.get(folderPath, fileName)

        // Check if the file exists
        return Files.exists(filePath)
    }
}
